packages/form/tests/integration/components/cf-field/label-test.js

```diff
@@ -48,12 +49,38 @@ module("Integration | Component | cf-field/label", function (hooks) {
   test("it marks optional fields as such", async function (assert) {
     assert.expect(2);
 
-    await render(hbs`<CfField::Label @field={{this.field}} />`);
+    await render(
+      hbs`<CfField::Label @field={{this.field}} @useMandatoryAsterisk={{this.useMandatoryAsterisk}} />`,
+    );
 
     assert.dom("label").hasText("Test");
 
     this.set("field.question.raw.isRequired", "false");
 
     assert.dom("label").hasText("Test (Optional)");
   });
+
+  test("it marks mandatory fields as such if useMandatoryAsterisk is configured", async function (assert) {
+    assert.expect(1);
+    this.set("useMandatoryAsterisk", true);
+
+    await render(
+      hbs`<CfField::Label @field={{this.field}} @useMandatoryAsterisk={{this.useMandatoryAsterisk}} />`,
+    );
+    this.set("field.question.raw.isRequired", "true");
+
+    assert.dom("label").hasText("Test *");
+  });
+
+  test("it doesnt mark optional fields if useMandatoryAsterisk is configured true", async function (assert) {
+    assert.expect(1);
+    this.set("useMandatoryAsterisk", true);
+
+    await render(
+      hbs`<CfField::Label @field={{this.field}} @useMandatoryAsterisk={{this.useMandatoryAsterisk}} />`,
+    );
+    this.set("field.question.raw.isRequired", "false");
+
+    assert.dom("label").hasText("Test");
+  });
```

Comment:
This is quite a verbose test for a really simple use-case. I'd do something like this:

```suggestion
  test("it marks mandatory fields as such if useMandatoryAsterisk is configured", async function (assert) {
    assert.expect(1);

    await render(
      hbs`<CfField::Label @field={{this.field}} @useMandatoryAsterisk={{true}} />`,
    );
    
    this.set("field.question.raw.isRequired", "true");
    assert.dom("label").hasText("Test *");
    
    this.set("field.question.raw.isRequired", "false");
    assert.dom("label").hasText("Test");
  });
```

Reply:
Tests look great now!